readthedocs/rtd_tests/tests/test_subprojects.py

```diff
@@ -56,3 +60,33 @@ def test_admin_of_subproject_can_add_it(self):
         self.assertEqual(
             [r.child for r in project.subprojects.all()],
             [subproject])
+
+
+@override_settings(PUBLIC_DOMAIN='readthedocs.org')
+class ResolverBase(TestCase):
+
+    def setUp(self):
+        with mock.patch('readthedocs.projects.models.broadcast'):
+            with mock.patch('readthedocs.projects.models.update_static_metadata'):
+                self.owner = create_user(username='owner', password='test')
+                self.tester = create_user(username='tester', password='test')
+                self.pip = get(Project, slug='pip', users=[self.owner], main_language_project=None)
+                self.subproject = get(Project, slug='sub', language='ja', users=[
+                                      self.owner], main_language_project=None)
+                self.translation = get(Project, slug='trans', language='ja', users=[
+                                       self.owner], main_language_project=None)
```

Comment:
Argument wrapping on lines 74-77 is confusing, due to list broken up over multiple lines.